nss-cohort-36/bangazon-api-illin-illusionists-api
# Ticket 18 - Show latest 20 products in Home

# Description

FEATURE: `Show latest 20 products in Home`

## Type of change

Please delete options that are not relevant.


## Testing Instructions
```shell
git fetch --all
git checkout gc-products-limit
python manage.py runserver
```

**GET latest 20**
1. Open Postman and check that you have ALL 22 items in the `http://localhost:8000/products` endpoint (**GET**)
using the headers:
```json
Content-Type application/json
Authorization Token c117003f825e8da8fbf38e65d46507224d29269c 
```
2. Open Postman and check that you get ONLY items  3 to 22 (latest items)  items in the `http://localhost:8000/products?limit=20
` endpoint (**GET**)
using the headers:
```json
Content-Type application/json
Authorization Token c117003f825e8da8fbf38e65d46507224d29269c 
```

## Checklist:
